### ISSUE TITLE
Capture region of the screen during screen recording

### ISSUE THREAD

**Issue by Jean-Bapt** [dahuapp/dahu#51]
It will be great to capture just a part of the screen instead of the entire screen (like greenshot does).
Then we can fix the dimensions of the capture and set the mouse position at the correct coordinate if it appears on the screen (no mouse otherwise).

**Comment by barraq** [comment]
Indeed this is an important feature.

We have two options here:
- ask the user to select a region before starting the screen recording.
- ask the user to select a region after the recording and crop recorded images.

I like the first one better.

**Comment by moy** [comment]
Duplicate of #13
